Makefile

```diff
@@ -59,7 +59,7 @@ prepare-external: $(PACKAGES) $(NUNIT_CONSOLE)
 clean:
 	-$(MSBUILD) $(MSBUILD_FLAGS) /t:Clean
 	-rm -Rf bin/$(CONFIGURATION) bin/Build$(CONFIGURATION) bin/Test$(CONFIGURATION) bin/XAIntegration$(CONFIGURATION)
-	-rm src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config
+	-rm -f src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config
```

Comment:
This change doesn't make sense. A `-` on the line tells `make` to ignore errors, so any errors from this line should *already* be ignored. I don't see what using `rm -f` will change.